README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# HW1
+
+# About me
+My name is **Chenyang He**. I am a MAPSS student concentrating in *Psychology*. 
+
+# Research Interest 
+I am currently working as an RA in Dr. Boaz Keysar's Multilingualism and Decision-Making Lab in the Psychology Department. 
+Here's a link to our lab page:
+[mdl](https://mdl.uchicago.edu/)
+
+We are dedicated to understanding the following interesting topics:
+* Bilingualism and Decision Making
+  * With millions of people worldwide using a foreign language everyday, how does language impact the way they make decisions? 
+* Accents
+  * What are the consequences of speaking with a foreign accent? 
+* Figurative Language
+  * How do people understand figurative language such as metaphors, idioms, irony and sarcasm? 
+
+# My best friend
+This is Peaches, my best friend during the quarantine.
+She makes me happy everyday :)
```

Comment:
Extremely nit picky advice, but maybe adding another line (more space) between the caption and the photo might balance the page a bit more. But not necessary at all.

Reply:
This is a valid point! You can use the "Preview" button at the top of your RStudio console to see what the rendered version of your report looks like, and make changes along these lines.